### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Found some misspellings with `typos` and my eyes.

### CODE REVIEW

File: wp-includes/sqlite/class-wp-sqlite-pdo-user-defined-functions.php [hunk 1]
```diff
@@ -446,11 +446,11 @@ public function isnull( $field ) {
...
 	 *
-	 * @param unknonw $expression the statement to be evaluated as true or false.
-	 * @param unknown $true statement or value returned if $expression is true.
-	 * @param unknown $false statement or value returned if $expression is false.
+	 * @param mixed $expression the statement to be evaluated as true or false.
+	 * @param mixed $true statement or value returned if $expression is true.
+	 * @param mixed $false statement or value returned if $expression is false.
 	 *
-	 * @return unknown
+	 * @return mixed
```
Comment (szepeviktor) [inline wp-includes/sqlite/class-wp-sqlite-pdo-user-defined-functions.php:453]: There is no type called `unknown`.

File: wp-includes/sqlite/class-wp-sqlite-translator.php [hunk 2]
```diff
@@ -1683,9 +1683,9 @@ private function preprocess_string_literal( $value ) {
 	private function preprocess_like_expr( &$token ) {
 		/*
 		 * This code handles escaped wildcards in LIKE clauses.
-		 * If we are within a LIKE experession, we look for \_ and \%, the
+		 * If we are within a LIKE expression, we look for \_ and \%, the
 		 * escaped LIKE wildcards, the ones where we want a literal, not a
-		 * wildcard match. We change the \ escape for an ASCII \x1a (SUB) character,
+		 * wildcard match. We change the \ escape for an ASCII \x1A (SUB) character,
```
Comment (szepeviktor) [inline wp-includes/sqlite/class-wp-sqlite-translator.php:1688]: Hexadecimal digit are usually written with uppercase letters.

**Review by aristath — APPROVED** [review]

**Comment by aristath** [comment]
Thank you for working on this @szepeviktor!

Merged 👍

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**